bin/e2e

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
-if [ "$KNAPSACK_PRO_TEST_SUITE_TOKEN_CYPRESS" = "" ]; then
-  KNAPSACK_PRO_ENDPOINT=https://api-disabled-for-fork.knapsackpro.com \
-    KNAPSACK_PRO_MAX_REQUEST_RETRIES=0 \
-    KNAPSACK_PRO_TEST_SUITE_TOKEN_CYPRESS=disabled-for-fork \
-    npx cypress run # run without parallelization
-else
-  npx knapsack-pro-cypress # parallelization is available
+
+read -p "Do you need to set up your end to end (E2E) testing database? Answer yes if this is your first time running E2E tests on your local machine or you need to recreate your E2E test database. (y/n) " -n 1 -r
```

Comment:
To make it easy for local e2e test development, the dev is prompted if they want to set up the e2e test database. This is typically a first time action or if something is messed up with their test database.